completions/ant

```diff
@@ -1,5 +1,30 @@
 # bash completion for ant and phing                        -*- shell-script -*-
 
+_ant_parse_targets()
+{
+    local line basedir
+
+    basedir=$(dirname "$1")
```

Comment:
It's about removing a matching suffix pattern. For details, see "Parameter Expansion" in the bash manual page.